### DISCUSSION TITLE
Random crashes

### DISCUSSION THREAD

**Discussion by tuncatunc** [coinfoundry/miningcore#918] [Q&A]
Miningcore pool randomly crashes without any log or trace.
I suspect it may be related to memory usage.
My current VPS is 8 core 16 GB RAM.

Before I upgrade my plan I'd like to know the possibilities of crashing.

I'll make it a service so it can restart and run at startup.

Thx

**Comment by AlphaX-Qortal** [comment]
I run 10 pools on a vps 6 cores 16gb ram 400gb ssd (wallets outsourced).
Not a single crash but i restart via cron every 24 hours.

**Comment by tuncatunc** [comment]
Recently I did fresh install on 32 GB VPS. So far no problem.
